Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM tomcat:8-jre8
+MAINTAINER Alexandros Sigaras <als2076@med.cornell.edu>, Fedde Schaeffer <fedde@thehyve.nl>
+LABEL Description="cBioPortal for Cancer Genomics"
+ENV APP_NAME="cbioportal" \
+    PORTAL_HOME="/cbioportal"
+#======== Install Prerequisites ===============#
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        git \
+        libmysql-java \
+        patch \
+        python \
+        python-jinja2 \
+        python-mysqldb \
+        python-requests \
+    # install new versions of these packages backported to Debian stable;
+    # Debian does not add new features or break backwards compatibility within
+    # a stable release, but for these dependencies we need versions that do.
+    && apt-get install -y --no-install-recommends -t jessie-backports \
+        maven \
+        openjdk-8-jdk \
+    && ln -s /usr/share/java/mysql-connector-java.jar "$CATALINA_HOME"/lib/ \
+    && rm -rf $CATALINA_HOME/webapps/examples \
+    && rm -rf /var/lib/apt/lists/*
+#======== Configure cBioPortal ===========================#
+COPY . $PORTAL_HOME
+WORKDIR $PORTAL_HOME
+EXPOSE 8080
+#======== Build cBioPortal on Startup ===============#
+CMD mvn -DskipTests clean install \
+     && cp $PORTAL_HOME/portal/target/cbioportal*.war $CATALINA_HOME/webapps/cbioportal.war \
+     && find $PWD/core/src/main/scripts/ -type f -executable \! -name '*.pl'  -print0 | xargs -0 -- ln -st /usr/local/bin \
```

Comment:
@fedde-s we have retained all good practices you followed on the variant you mentioned.

This dockerfile aims to support docker automated builds regardless of the git tag or release and is meant to blend in with the code.
In short, every time there is a tag release in github there will be a docker equivalent version build automatically in docker hub using github triggers.

This allows for full transparency on what the image contains, vetted by Docker itself as it is build in their cloud infrastructure with zero interventions.